src/engine/specification.md

```diff
@@ -219,7 +219,7 @@ Payload validation process consists of validating a payload with respect to the
 
 3. Client software **MUST** validate a payload according to the block header and execution environment rule set with modifications to these rule sets defined in the [Block Validity](https://eips.ethereum.org/EIPS/eip-3675#block-validity) section of [EIP-3675](https://eips.ethereum.org/EIPS/eip-3675#specification):
   * If validation succeeds, the response **MUST** contain `{status: VALID, latestValidHash: payload.blockHash}`
-  * If validation fails, the response **MUST** contain `{status: INVALID, latestValidHash: validHash}` where `validHash` is the block hash of the unique ancestor of the invalid payload satisfying the following two conditions:
+  * If validation fails, the response **MUST** contain `{status: INVALID, latestValidHash: validHash}` where `validHash` is either `null`, or the block hash of the unique ancestor of the invalid payload satisfying the following two conditions:
```

Comment:
```suggestion
  * If validation fails, the response **MUST** contain `{status: INVALID, latestValidHash: validHash}` where `validHash` is either `0x`, or the block hash of the unique ancestor of the invalid payload satisfying the following two conditions:
```

I suggest we confirm that this entire clarification goes into the spec, I just want to double check with teams on ACD. And then decide how to encode `null` -- either option is fine with me

Reply:
Since we don't really care about compression, I would prefer a well defined string that describes the problem.  While this won't change anything functionally, it makes it easier to understand what is happening for someone who may not be intimately familiar with the specification.

Reply:
The problem with `'syncing'` is that `latestValidHash` field is of `DATA` type and it either should be a valid `DATA` value or `null`.

Reply:
Please encode `null` as JSON `null` :)

Reply:
Hmm, Is the issue that not all languages support union types?